invenio_rdm_records/resources/serializers/ui/schema.py

```diff
@@ -64,6 +64,23 @@ def apply_idx(affiliation):
     }
 
 
+def localize_vocabulary_list(field_name, key, attr, obj):
+    """Localize vocabulary."""
+    field_data = attr.get("metadata").get(field_name)
```

Comment:
Is it worth improving this to receive a dotted path? e.g. get`metadata.languages` as `field_name` and process it according to the dots